projects/storyteller/webapp/src/app/components/user-info/user-info.component.html

```diff
@@ -0,0 +1,13 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Could you put it to `index.html`

Reply:
BTW do we really need it?

Reply:
done.
i'm not quite sure about that because in font-awesome have twitter/facebook icons
and i thought to do some connection between sotryteller to github.
delete it?